docusaurus/docs/dev-docs/configurations/database.md

```diff
@@ -535,3 +539,17 @@ $ GRANT ALL ON SCHEMA public TO my_strapi_db_user;
 ```
 
 :::
+
+### Alternative database drivers
+
+Strapi attempts to load database drivers for each client based on which packages have been added to the project.
+
+| client   | order of driver selected when available  |
+| -------- | ---------------------------------------- |
+| mysql    | mysql2, mysql                            |
```

Comment:
Would it be worth linking to the packages on github or npm like we do with knex?

Reply:
I think so, yes! :)